src/components/BlogsSlider/index.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import Slider from "react-slick";
+import { useTranslation } from "react-i18next";
+import useStyles from "./style";
+import {
+  Button,
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+} from "@material-ui/core";
+import { VIEW_PORT } from "../../constants";
+import CustomTag from "../CustomTag";
+import ReactMarkdown from "react-markdown";
+
+interface BlogsSliderProps {
+  recommendedBlogs: any;
+}
+
+const BlogsSlider: React.FC<BlogsSliderProps> = ({recommendedBlogs}) => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+  const mobileBreakpoint = VIEW_PORT.MOBILE_BREAKPOINT;
+
+  const SampleNextArrow = (props: any) => {
+    const { className, style, onClick } = props;
+    return (
+      <div
+        className={className}
+        style={{ ...style, display: "block" }}
+        onClick={onClick}
+      >
+        <img
+          src="../Images/svg/right_arrow.svg"
+          alt={t("home.adaptorsTestimonials.nextArrowAlt")}
+        />
+      </div>
+    );
+  };
+
+  const SamplePrevArrow = (props: any) => {
+    const { className, style, onClick } = props;
+    return (
+      <div
+        className={className}
+        style={{ ...style, display: "block" }}
```

Comment:
Can we try to avoid inline styles ?

Reply:
This is within the `react-slick` plugin. Styling with class name is not supported. Arrows need to be customized only through inline CSS https://react-slick.neostack.com/docs/example/custom-arrows